docs/topics/proc_deploying-a-node-js-application-to-standalone-rhel.adoc

```diff
@@ -0,0 +1,45 @@
+[id='deploying-a-node-js-application-to-standalone-rhel_{context}']
+= Deploying a {Node} application to standalone Red Hat Enterprise Linux
+
+Deploy a {Node} application to Red Hat Enterprise Linux.
+
+.Prerequisites
+
+* A {Node} application.
+* npm 6.4.1 installed
+* RHEL 7 installed.
+* {Node} installed
+x
+
+.Procedure
+
+. Navigate to the directory with your application project:
++
+[source,bash,subs="attributes+",options="nowrap"]
+----
+$ cd myApp
+----
+
+. If you have specified additional dependencies in the `package.json` file of your project, ensure that you install them before running your applications:
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ npm install
+----
+
+. Start your application.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ node app.js
+Server running at http://localhost:8080
+----
+
+.Verification steps
+
+. Use `curl` or your browser to verify your application is running at `http://localhost:8080`
```

Comment:
Again, this makes an assumption about the application being deployed using port 8080 for the web service.

Reply:
Good point. Somehow I completely overlooked this.
The port configuration should be shown in the `app.js` example.